packages/generator-joplin/generators/app/templates/api/JoplinImaging.d.ts

```diff
@@ -34,6 +49,8 @@ export default class JoplinImaging {
     private cacheImage;
     createFromPath(filePath: string): Promise<Handle>;
     createFromResource(resourceId: string): Promise<Handle>;
+    createFromPdfPath(path: string, options?: CreateFromPdfOptions): Promise<Handle[]>;
+    createFromPdfResource(resourceId: string, options?: CreateFromPdfOptions): Promise<Handle[]>;
```

Comment:
Consider adding a `getPDFInfoFromPath` or `getPDFInfoFromResource` here to allow getting PDF metadata, text, etc.

We don't need to deal with caching for now. It can be a performance optimization for later, if necessary.

Reply:
`getPdfInfoFromPath` and `getPdfInfoFromResource` have been added.